kmod/igb_main.c

```diff
@@ -134,6 +134,9 @@ static void igb_dma_err_timer(unsigned long data);
 static void igb_watchdog_task(struct work_struct *);
 static void igb_dma_err_task(struct work_struct *);
 /* AVB specific */
+#ifdef HAVE_PTP_1588_CLOCK
```

Comment:
Why is this conditional? Don't all devices covered by this driver support PTP_1588_CLOCK?